.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = stock-logistics-workflow
 	url = git@github.com:Vauxoo/stock-logistics-workflow.git
 	branch = 12.0
+[submodule "queue"]
+	path = queue
+	url = git@github.com:OCA/queue.git
```

Comment:
OCA or vauxoo?

Reply:
this is just for test purposes, if correct, then I'll update vauxoo/queue since it's  201 commits behind OCA:12.0.